<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jzys2_widget</class>
 <widget class="QWidget" name="jzys2_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>607</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>645</width>
    <height>607</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>645</width>
    <height>607</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>101</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>进制：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>操作数(1):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>运算符:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>进制：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>操作数(2):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>比较大小：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>运算结果：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>346</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>二进制</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>十六进制</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>八进制</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="text">
       <string>十进制</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>346</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>二进制</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>十六进制</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string>八进制</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="text">
       <string>十进制</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_yunsuan">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>运算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_qingkong">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bijiao_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>比较</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_gongyueshu">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>公约数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gongbeishu">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>公倍数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chu">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cheng">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_jian">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_jia">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bijiao">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>55</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>比较</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>十进制</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>二进制</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>十六进制</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>490</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>490</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>八进制</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
